services/api-server/src/simcore_service_api_server/api/dependencies/services.py

```diff
@@ -0,0 +1,34 @@
+""" Dependences with any other services (except webserver)
+
+"""
+from typing import Callable, Type
+
+from fastapi import HTTPException, Request, status
+
+from ...utils.client_base import BaseServiceClientApi
+
+
+def get_api_client(client_type: Type[BaseServiceClientApi]) -> Callable:
+    """
+        Retrieves API client from backend services EXCEPT web-server (see dependencies/webserver)
+
+        Usage:
+
+            director_client: DirectorApi = Depends(get_api_client(DirectorApi)),
+            catalog_client: CatalogApi = Depends(get_api_client(CatalogApi)),
+            storage_client: StorageApi = Depends(get_api_client(StorageApi)),
+    """
+    assert issubclass(client_type, BaseServiceClientApi)  # nosec
```

Comment:
This logic is not intended to be part of the code but a check while development (I mentioned many it before that I like using defensive programming). It is actually [removed in production](https://github.com/ITISFoundation/osparc-simcore/blob/master/services/api-server/Dockerfile#L113) and the comment ``# nosec `` is to avoid codeclimate warning me about it as you did :-)